lib/main.js

```diff
@@ -47,6 +48,19 @@ const fixPathString = (pathString, fileDir, projectDir) => {
   return pRstring;
 };
 
+const resolvePath = (fullPath) => {
+  const paths = fullPath.split(';');
+  if (paths.length === 1) {
+    return paths[0];
+  }
+  for (let i = 0; i < paths.length; i += 1) {
```

Comment:
I would just replace the first 5 lines of this with:
```js
fullPath.split(';').forEach((path) => {
```

The only difference from the existing case is the additional `existsSync` call in the case where there was only one result.